R/fda-table_32.R

```diff
@@ -0,0 +1,69 @@
+# Pre-processing
+
+library(tidyverse)
+library(scda)
+library(tern)
+
+# Safety population
+adsl <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "adsl") %>%
+  tern::df_explicit_na() %>%
+  dplyr::filter(SAFFL == "Y") %>%
+  dplyr::select(STUDYID, USUBJID, SAFFL, ARM, ACTARM)
```

Comment:
Not necessary to select columns in the dataset(s).
```suggestion
  dplyr::filter(SAFFL == "Y")
```